manifests/init.pp

```diff
@@ -203,6 +203,8 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
+# $max_speed::                  Maximum bandwidth used per download in bytes/sec from CDN
```

Comment:
Is this setting only relevant for yum importer? I think the top level variable name should indicate it, maybe `$yum_max_seed`.  

This also applies to all yum repos synced by pulp, right? So I'd avoid saying CDN, which in the context of pulp probably only applies to Red Hat.